tests/print_status/run.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+function run() {
+    run_sql_file $cur/data/db.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1
+
+    # in load stage, the dumped file split into 14 insert segments, we slow down 14 * 100 ms
+    # in sync stage, there are approximately 530~550 binlog events, we slow down 3 * 530 ms
```

Comment:
```suggestion
    # in sync stage, there are approximately 530~550 binlog events, we slow down 530 * 3 ms
```
or update L12 to make them unified?